doc/classes/RefCounted.xml

```diff
@@ -37,7 +37,7 @@
 			<return type="bool" />
 			<description>
 				Decrements the internal reference counter. Use this only if you really know what you are doing.
-				Returns [code]true[/code] if the decrement was successful, [code]false[/code] otherwise.
+				Returns [code]true[/code] if the decrement killed the object, [code]false[/code] otherwise.
```

Comment:
We don't really use `kill` terminally for this, but `free`.
```suggestion
				Returns [code]true[/code] if the decrement freed the object, [code]false[/code] otherwise.
```

Reply:
You can edit it, I truly dont mind I just want the issue resolved